dem_stitcher/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class NoDEMCoverage(Exception):
+    "Throw this exception if extent/bounds does not cover any available DEM tiles"
+    pass
```

Comment:
```suggestion
    """Throw this exception if extent/bounds does not cover any available DEM tiles"""
```
As long as you have _docstrings_, you don't need the `pass`. For example:
https://github.com/ASFHyP3/hyp3-sdk/blob/develop/hyp3_sdk/exceptions.py#L7-L8